e2e/nomostest/new.go

```diff
@@ -438,6 +441,23 @@ func setupTestCase(nt *NT, opts *ntopts.New) {
 	if *e2e.GitProvider == e2e.Local {
 		nt.portForwardGitServer()
 	}
+	// set up port forward if using in-cluster registry server
+	if *e2e.RegistryProvider == e2e.Local {
+		nt.portForwardRegistryServer()
+	}
+	// Images created by the tests should not be persisted after the test finishes.
+	// Cleanup all images before the port forward gets torn down.
+	nt.T.Cleanup(func() {
+		for _, image := range nt.ociImages {
+			remoteURL, err := nt.RegistryProvider.OCIPushURL(image.Name)
+			if err != nil {
+				nt.T.Fatal(err)
+			}
+			if err := image.Delete(remoteURL); err != nil {
+				nt.T.Error(err)
+			}
+		}
```

Comment:
should we clear ociImages after?

Reply:
It isn't strictly necessary since the next test will start with an empty array, but I added a line to clear it.